ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/mapping/properties/PropertyHandler.java

```diff
@@ -0,0 +1,35 @@
+package no.difi.meldingsutveksling.ks.mapping.properties;
+
+import com.google.common.base.Objects;
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.ks.mapping.Handler;
+
+public class PropertyHandler<T> implements Handler<T> {
+    private final IntegrasjonspunktProperties properties;
+
+    public PropertyHandler(IntegrasjonspunktProperties properties) {
+        this.properties = properties;
+    }
+
+    @Override
+    public T map(T builder) {
+        return builder;
+    }
+
+    public IntegrasjonspunktProperties getProperties() {
+        return properties;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof PropertyHandler)) return false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00122)